src/cells/StochasticCorrector.js

```diff
@@ -0,0 +1,63 @@
+
+import Cell from "./Cell.js" 
+
+/**
+ * Implements a basic holder for a model with two internal products,
+ * which can be stochastically divided between daughter cells. 
+ */
+class StochasticCorrector extends Cell {
```

Comment:
Q: How generic do you think this model is? 

If it is just here to support the provided example, consider removing it from the build and defining this class in the script of the example itself; this prevents the main library from becoming too bloated. you can do something similar as in [this tutorial in the manual](https://artistoo.net/manual/custommodules.html).

Reply:
I agree, I'll either try to generalise this example or move it to the example config

Reply:
Okay great! I think it's fine to keep the example as it is; it is perhaps also useful to have an example where an extra class is defined (letting users know how to do this without having to overwrite the build).

Reply:
Ah right, I saw this slightly too late, and rewrote the Stochastic corrector class to a Divider with an arbitrary set of evolvable internal products. This was partially because I also wanted to provide an example of Cell subclassing in the codebase, but this could also still be moved to the example

Reply:
I just saw! I'll leave it up to you: if you think more people will use this functionality, then just leave it in the build. If this is still relatively specific to your work, put it in the example for now (we can always add it to the main build later if people request this, but if you already think people will use this then we might as well just do it now).